README.md

```diff
@@ -1,221 +1,10 @@
 # Matrix-appservice-slack
-A Matrix &lt;--> Slack bridge
+A Matrix &lt;--&gt; Slack bridge
```

Comment:
A screenshot showing the bridge doing all it's things would be awesome too! [like this](https://github.com/Half-Shot/matrix-appservice-discord#matrix-discord-bridge)